packages/grafana-prometheus/src/components/monaco-query-field/monaco-completion-provider/data-provider.ts

```diff
@@ -0,0 +1,117 @@
+import { HistoryItem } from '@grafana/data';
+// @ts-ignore
+import type { Monaco } from '@grafana/ui';
```

Comment:
Do we need this? What happens when we remove this import?

Reply:
If we remove this, it breaks the TSDoc [`@link`](https://tsdoc.org/pages/tags/link) below.

With Monaco type import:
![tsdoc link Monaco Range](https://github.com/grafana/grafana/assets/5732000/5db90b74-84ba-47ee-b737-2ac9eed8d650)

Without:
![broken tsdoc link Monaco Range](https://github.com/grafana/grafana/assets/5732000/24e17f90-5bfa-4186-81cd-cf82ea20d08d)

If you feel the `@link` is of questionable value, I'm happy to remove it.

Reply:
Do you mind commenting on this over there? It will help us in the future I believe.

Reply:
Certainly! Updated in https://github.com/grafana/grafana/pull/85396/commits/7b9510c29586c8402cd87872b99f5d580d66e44e.